app/models/piece.rb

```diff
@@ -1,2 +1,8 @@
 class Piece < ActiveRecord::Base
+  belongs_to :game
+
+  def self.types
+    %w(King Queen Rook Bishop Knight Pawn)
```

Comment:
# We will need a way to know which animals
    # will subclass the Animal model
    def self.races
      %w(Lion WildBoar Meerkat)
    end

FYI for whatever reason he replaced "type" with "races".